app/views/shared/_sidebar.html.erb

```diff
@@ -38,10 +61,13 @@
       <li>
         <%= link_to "Community", "/community", sidebar_link_options("community") %>
       </li>
+      <!-- <li>
```

Comment:
this seems like old code that can be removed?

Reply:
@chotuaashiq don't forget to remove these lines if they are not relevant. I will give you some more feedback later when I get the time to have a proper look :)